pages/components/invoice-card.tsx

```diff
@@ -11,23 +12,34 @@ import {
   Image,
 } from "@nextui-org/react";
 import axios from "axios";
-import { CheckIcon, ClipboardIcon } from "@heroicons/react/24/outline";
+import {
+  BoltIcon,
+  CheckIcon,
+  ClipboardIcon,
+  EnvelopeIcon,
+} from "@heroicons/react/24/outline";
 import { CashuMint, CashuWallet, getEncodedToken } from "@cashu/cashu-ts";
 import { getLocalStorageData } from "./utility/nostr-helper-functions";
 import { nip19 } from "nostr-tools";
 import { ProductData } from "./utility/product-parser-functions";
-import { DisplayCostBreakdown } from "./utility-components/display-monetary-info";
+import {
+  DisplayCostBreakdown,
+  formatWithCommas,
+} from "./utility-components/display-monetary-info";
+import { SHOPSTRBUTTONCLASSNAMES } from "../components/utility/STATIC-VARIABLES";
 import { captureInvoicePaidmetric } from "./utility/metrics-helper-functions";
 
-export default function CheckoutCard({
+export default function InvoiceCard({
   productData,
 }: {
   productData: ProductData;
 }) {
   const router = useRouter();
   const { pubkey, currency, totalCost } = productData;
   const pubkeyOfProductBeingSold = pubkey;
-  const { decryptedNpub, relays, mints } = getLocalStorageData();
+  const { npub, decryptedNpub, relays, mints } = getLocalStorageData();
+
+  const [paymentCard, setPaymentCard] = useState(false);
```

Comment:
I think we need a better name for this cause the current name is ambiguous. Maybe try something like showPaymentQRCode or soemthing along those lines

Reply:
Updated it to invoiceCard